src/grouping.js

```diff
@@ -0,0 +1,17 @@
+const { getCardNumber } = require("./getCardNumber");
+
+const numbers = hand.map(getCardNumber);
+
+numbers.sort(function (a, b) {
+  return a - b;
+});
```

Comment:
Shortened suggestion
```suggestion
numbers.sort((a, b) => a - b);
```